NEWS.md

```diff
@@ -54,6 +54,12 @@ Breaking changes
     is now divided among the fields `code`, `slotnames`, `slottypes`, `slotflags`,
     `gensymtypes`, `rettype`, `nargs`, and `isva` in the `LambdaInfo` type ([#15609]).
 
+  * `pmap` keyword arguments `err_retry=true` and `err_stop=false` are depricated.
+    `pmap` no longer retries or returns `Exception` objects in the result collection.
```

Comment:
No strong opinion on this. My feeling is that if it was "neither retries", then it would be "nor returns". I believe that in the "does not verb or verb" case, there is no hard rule about "or" or "nor". When I read it out aloud as written, it sounds better to me with "or" rather than "nor".  However, I'm Australian, so what sounds normal to me would make an Englishman cringe! Happy to change it if you feel strongly ...